File: hanghae-cinema/domain/src/main/java/org/example/domain/ScreeningSchedule.java
package org.example.domain;

import lombok.Builder;
import lombok.Getter;

import java.time.LocalDateTime;

@Getter
@Builder
public class ScreeningSchedule {
    private Long id;
    private Screening screening;
    private LocalDateTime startTime;
    private LocalDateTime endTime;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private String createdBy;
    private String updatedBy;
}

File: movies/src/main/kotlin/com/redis/domain/ScreeningSchedule.kt
package com.redis.domain

import com.redis.utils.BaseEntity
import jakarta.persistence.*
import java.time.LocalDateTime

@Table(name = "screening_schedule")
@Entity
data class ScreeningSchedule(

    @Column(name = "start_time")
    val startTime: LocalDateTime,

    @Column(name = "end_time")
    val endTime: LocalDateTime,

    @ManyToOne
    @JoinColumn(name = "movie_id")
    val movie: Movie? = null,

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "screening_schedule_id")
    val id: Long? = null
): BaseEntity() {
}